daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -1819,6 +1819,7 @@ class EngineTest
     }
   }
 
+  // TEST_EVIDENCE: Semantics: Contract key behaviour
   "contract key" should {
```

Comment:
did you intend to drop these here? It looks like they are duplicated now.

Reply:
I totally did mean to drop these! Thanks for catching this. Dropped now.